settings.py

```diff
@@ -3,7 +3,24 @@
 
 logger = logging.getLogger(__name__)
 
-LOGGING_FORMAT = "%(asctime)s|%(levelname)s: sdx-sequence: %(message)s"
+LOGGING_FORMAT = "%(message)s"
 LOGGING_LEVEL = logging.getLevelName(os.getenv('LOGGING_LEVEL', 'DEBUG'))
 
-MONGODB_URL = os.getenv("MONGODB_URL", "mongodb://localhost:27017")
+
+def _get_value(key):
+    value = os.getenv(key)
+    if not value:
+        raise ValueError("No value set for " + key)
+    else:
+        return value
+
+try:
+    DB_HOST = _get_value("SDX_SEQUENCE_POSTGRES_HOST")
+    DB_PORT = _get_value('SDX_SEQUENCE_POSTGRES_PORT')
+    DB_NAME = _get_value('SDX_SEQUENCE_POSTGRES_NAME')
+    DB_USER = _get_value('SDX_SEQUENCE_POSTGRES_USER')
+    DB_PASSWORD = _get_value('POSTGRES_PASSWORD')
+    DB_URL = 'postgres://{}:{}@{}:{}/{}'.format(DB_USER, DB_PASSWORD, DB_HOST, DB_PORT, DB_NAME)
+
+except ValueError as e:
```

Comment:
Can the value error that was thrown be logged as well - underrstanding which envvar is missing would help.